frontend/actions/auth.js

```diff
@@ -134,13 +134,18 @@ export const logInWithGoogle = googleToken => {
     }
 };
 
-export const register = (login, password) => {
+export const register = (login, password, passwordRepeat) => {
     return async dispatch => {
         console.log('registering...');
 
         const name = login;
         password = SHA256(password).toString();
+        passwordRepeat = SHA256(passwordRepeat).toString();
 
+        if (password != passwordRepeat) {
```

Comment:
jeśli już tutaj sprawdzasz czy te hasła sa takie same to nie sądzę aby była koniecznośc przesylania ich do serwera ;)

Reply:
Poprawione, brak przesyłania passwordRepeat na serwer